.buildkite/linux_aarch64.rayci.yml

```diff
@@ -0,0 +1,33 @@
+group: Linux aarch64 tests
+depends_on:
+  - manylinux-aarch64
+  - oss-ci-base_build-aarch64
+  - forge-aarch64
+steps:
+  # block on premerge
+  - block: "Run Linux aarch64 tests"
+    if: build.env("BUILDKITE_PIPELINE_ID") == "0189942e-0876-4b8f-80a4-617f988ec59b"
+
```

Comment:
could you move the image building jobs in `build.rayci.yaml` file here too?

Reply:
done